.travis.yml

```diff
@@ -29,3 +29,4 @@ script:
     done
 
   - make all
+  - make generate_k8s
```

Comment:
Can we give a name like `kubernetes_object_yaml`?

Reply:
Ack.